style.css

```diff
@@ -10,402 +10,713 @@ body {
   font-family: 'Poppins', sans-serif;
 }
 
-@media (min-width: 375px) {
-  head {
-    margin-top: 10px;
+head {
+  margin-top: 2.6667%;
+}
+
+header {
+  padding: 4.2667% 6.4% 0 6.4%;
+  background-color: #fff;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  background-color: #e5e5e5;
+}
+
+.content {
+  background-image: url("header-shapes mobile.png");
+  background-size: 100%;
+  background-position: cover;
+  background-repeat: no-repeat;
+  background-color: #fff;
+  padding: 0;
+  border-radius: 0 0 0 25.62%;
+}
+
+.menuBar {
+  display: none;
+}
+
+.menu a {
+  width: 20.28%;
+  height: 5.33%;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.125rem;
+  line-height: 5.33%;
+  color: #6070ff;
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.menu {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.menu img {
+  width: auto;
+  height: auto;
+}
+
+.intro h1 {
+  font-family: 'Poppins', sans-serif;
+  font-size: 2.5rem;
+  line-height: 3.25rem;
+  padding: 29% 6.4% 3.2% 6.4%;
+  font-weight: 700;
+  color: #172b4d;
+}
+
+.intro p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.25rem;
+  padding: 0 6.4% 3.2% 6.4%;
+  color: #344563;
+}
+
+.connect {
+  font-family: 'Poppins', sans-serif;
+  color: #7f8cff;
+  font-weight: 500;
+  font-size: 1rem;
+  padding: 0 6.4% 3.2% 6.4%;
+}
+
+.social {
+  display: flex;
+  align-items: center;
+  gap: 4.27%;
+  cursor: pointer;
+}
+
+.content .social {
+  padding: 0 6.4% 36.3% 6.4%;
+}
+
+.social ul {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  list-style: none;
+  gap: 5.07vw;
+}
+
+/* -------------------------------------middle ------------------------- */
+.middle {
+  background-color: #e5e5e5;
+  padding-top: 30.4%;
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 6.1vw;
+  padding-bottom: 30.4%;
+}
+
+.card {
+  background-color: #fff;
+  font-family: 'Poppins', sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #dfe1e6;
+  border-radius: 1.11vw;
+  margin: 6.4%;
+}
+
+.dimg {
+  display: none;
+}
+
+.mimg {
+  display: block;
+}
+
+.card img {
+  margin: 6.4%;
+  max-width: 100%;
+}
+
+.card h4 {
+  font-weight: 700;
+  font-size: 2rem;
+  line-height: 2.75rem;
+  color: #172b4d;
+  margin: 2.668% 6.4%;
+}
+
+.first {
+  list-style-type: none;
+}
+
+.list {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: center;
+  list-style-type: none;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 0.81rem;
+  line-height: 1rem;
+  color: #7a869a;
+  margin: 2.668% 6.4%;
+}
+
+.list li {
+  margin-right: 3.2%;
+}
```

Comment:
- [ ] [OPTIONAL] Nice job so far. However certain parts of your mobile version is still not responsive. I  am making this optional because the previous reviewer did not draw you attention to that, but it is very important to work on it before moving on. The image in the project card shrinks and get smaller than the original design as shown below. Kindly check and fix it. 

### Your design 

![Screenshot from 2023-06-09 16-09-42](https://github.com/Porkalaikarthika/Portfolio/assets/65233860/79e1282d-a5b2-4dea-bad0-87951ed0061e)


### Actual design 

![Screenshot from 2023-06-09 16-09-20](https://github.com/Porkalaikarthika/Portfolio/assets/65233860/d50aa5ab-5952-42b1-aec2-ff5cdc3a49cf)